File: ex1/plotData.m
function plotData(x, y)
%   Plots data points
%   x = population and y = profit.
    figure; % open a new figure window
    plot(x, y, 'rx', 'MarkerSize', 10); % Plot the data
    ylabel('Profit in $10,000s');
    xlabel('Population of City in 10,000s');
end

File: ex1/plotData.py
import matplotlib.pyplot as plt
def plotData(x, y):
#   Plot data points
#   x = population, y = profit
    plt.figure
    plt.plot(x, y, 'rx', markersize=10)
    plt.ylabel('Profit in $10,000s');
    plt.xlabel('Population of City in 10,000s');
    plt.show(block=False)
    return

